w2/num.py

```diff
@@ -1,8 +1,8 @@
 
-class NUM:
+class Num:
```

Comment:
Should it be NUM as discussed?

Reply:
Yes. I will update it with the next commit.